src/components/Label/__stories__/label.mdx

```diff
@@ -93,36 +93,38 @@ A label indicates the status of an item.
       },
       negative: {
         component: [
-          <div className="monday-storybook-label_block">
+          <div className="monday-storybook-label_block" key="uniqueKey1">
```

Comment:
I think we should better solve it in the `ComponentRules` level (in [vibe-storybook-components](https://github.com/mondaycom/vibe-storybook-components)) instead, it will save us the next to give a unique key for each child in every component instance

Reply:
oh okay, I think that is something I can look at I saw it as mdx and thought its not rules specific, will check and try to implement

Reply:
@talkor  so should I revert the changes for this as the component fix would be in different repo and lib components?

Reply:
reverted the change.